truss/base/trt_llm_config.py

```diff
@@ -91,40 +91,26 @@ def check_max_beam_width(cls, v: int):
         return v
 
 
-class TrussTRTLLMServingConfiguration(BaseModel):
-    engine_repository: str
-    tokenizer_repository: str
-    tensor_parallel_count: int = 1
-    pipeline_parallel_count: int = 1
+class TrussTRTLLMRuntimeConfiguration(BaseModel):
+    kv_cache_free_gpu_mem_fraction: float = 0.9
+    enable_chunked_context: bool = False
+    num_draft_tokens: Optional[int] = None
```

Comment:
Worth adding a comment explaining the effect of setting this value. e.g. this is only valid for spec dec I imagine and the value will be ignored when set for non-specdec?